### ISSUE TITLE
Ipyvolume cannot be imported in Jupyterlite: KeyError: 'emscripten'

### ISSUE THREAD

**Issue by MarcSkovMadsen** [widgetti/ipyvolume#427]
## Reproduce

- Open https://jupyter.org/try-jupyter/lab/
- Open a new notebook
- run a cell containing

```python
import piplite
await piplite.install('ipyvolume')
```

- run a cell containing

```python
import ipyvolume
```

![image](https://user-images.githubusercontent.com/42288570/216800094-70d61de8-297a-4320-8e9b-3ef19033a6ae.png)

```bash
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Cell In[2], line 1
----> 1 import ipyvolume

File /lib/python3.10/site-packages/ipyvolume/__init__.py:6
      4 from ipyvolume import styles  # noqa: F401
      5 from ipyvolume import examples  # noqa: F401
----> 6 from ipyvolume import datasets  # noqa: F401
      7 from ipyvolume import embed  # noqa: F401
      8 from ipyvolume.widgets import *  # noqa: F401, F403

File /lib/python3.10/site-packages/ipyvolume/datasets.py:13
      9     from urllib.request import urlretrieve  # py3
     11 import numpy as np
---> 13 osname = dict(darwin="osx", linux="linux", windows="windows")[platform.system().lower()]
     14 data_dir = os.path.expanduser("~/.ipyvolume/datasets")
     15 if not os.path.exists(data_dir):

KeyError: 'emscripten'
```

## Context

I am trying to improve the user experience of running [Panel](https://panel.holoviz.org/) in [its custom build](https://panel.holoviz.org/user_guide/Running_in_Webassembly.html#setting-up-jupyterlite) of Jupyterlite called [Panelite](https://panelite.holoviz.org/) in [Panel #4388](https://github.com/holoviz/panel/pull/4388).

Now I am working with the notebook `reference/panes/IPyWidget.ipynb` which needs to `import ipyvolume`. But it fails.

**Comment by scrgiorgio** [comment]
@MarcSkovMadsen 

don't know if it helps. But this is what I am doing in JupyterLite:

```
# cell 1
%pip install -q ipywidgets plotly ipyvolume ipython_genutils

# cell 2
def PatchIPyVolume(filename="/lib/python3.10/site-packages/ipyvolume/datasets.py"):
    # osname = dict(darwin="osx", linux="linux", windows="windows")[platform.system().lower()]
    with open(filename,"r") as f: lines=f.readlines()
    lines=["osname = platform.system().lower()\n" if line.startswith("osname =") else line for line in lines]
    with open(filename,"w") as f: f.writelines(lines)
PatchIPyVolume()

# cell3
import ipyvolume as ipv
```

**Comment by maartenbreddels** [comment]
This should be fixed in 0.6.1, thanks for reporting Marc!

Nice patch @scrgiorgio

**Comment by MarcSkovMadsen** [comment]
Thx all.
